### DISCUSSION TITLE
How to build the win-x64 executable?

### DISCUSSION THREAD

**Discussion by christescu** [tautcony/ISTA-Patcher#37] [Q&A] [answered]
In order to build the executable I use: _dotnet publish --self-contained true_ in VSCode.
The executable works good, however size is ten time bigger compared to the released file from assets.
What method is used to get such small size exe file?

**Comment by tautcony** [comment]
Well... the answer is in your command argument, `--self-contained` will include all the necessary dependencies to make it run without installing dotnet runtime.

**Comment by christescu** [comment]
Yes, that's clear, but how should I proceed to get exactly the released Ista-patcher.exe file  from provided source code?

**Comment by tautcony** [comment, marked as answer]
All released files are built by github action, check command here:

https://github.com/tautcony/ISTA-Patcher/blob/4ffd9f63c0e0753f1fe3343f077b307ec3419915/.github/workflows/build.yml#L51
